core/templates/dev/head/pages/exploration_player/HintAndSolutionModalService.js

```diff
@@ -40,7 +42,8 @@ oppia.factory('HintAndSolutionModalService', [
               $scope.isHint = true;
               $scope.hint = HintsAndSolutionManagerService.displayHint(index);
               var contentIdsToAudioTranslations =
-                ExplorationEngineService.getContentIdsToAudioTranslations();
+                ExplorationPlayerStateService.getCurrentEngineService().
```

Comment:
Again just do ExplorationPlayerStateService.getContentIdsToAudioTranslations().

See ["Law of Demeter"](https://en.wikipedia.org/wiki/Law_of_Demeter) -- don't talk to strangers.

Reply:
Done.